cep-XXXX.md

```diff
@@ -88,13 +104,46 @@ valid `subdirs`, with one exception: `noarch` MUST be rejected as a valid platfo
 In the absence of the field, the single value MUST be assumed to correspond to a user-specified
 value, falling back to the running platform value . For example, running on Linux x86_64 the `platforms` would default to `[linux-64]`.
 
-### `category`
+#### `category`
 
 Optional, `str`.
 
 A free-form field where users can annotate the type of dependencies handled in the environment
 file. For example, `dev` or `test`.  Mostly used by tools that generate lockfiles.
 
+### Preprocessing selectors
+
+`environment.yml` files MAY specify per-line filters that contain an expression that MUST be
+evaluate to a boolean. If `true`, the expression MUST be removed and the value MUST be kept in the
+document. If `false`, both the expression and the value MUST be removed.
+
+#### Comment-based selectors
+
+The syntax for these selectors MUST be in the form of a line comment with the expression
+between square brackets: `# [<expression>]`. The space between `#` and the opening square bracket
+is optional but recommended. Once the selectors have been processed, the resulting document MUST
+be parsable as YAML.
+
+The expressions MUST be composed of one or more boolean variables, and can be chained with `and`
+and `or` logic operators. Precedence can be specified with parentheses.
+
+The supported boolean variables are the OS- and architecture-specific values collected in
+[conda-build's 25.1.x documentation][selectors], plus `osx64` (equivalent to `osx and x64`). More
+specifically,`py`, `py*`, `np` and `build_platform` are NOT supported.
+
+These are used by `conda-lock`.
+
+#### Dictionary-based selectors
```

Comment:
These are used by micromamba?

Reply:
Yes, only docs about it are in https://github.com/mamba-org/provision-with-micromamba#extra-specs, but you can also see it in the code https://github.com/mamba-org/mamba/blob/26785a6b50bea434cf8b7d964f92f0f407b34b0a/libmamba/tests/src/core/test_env_file_reading.cpp#L18. Let me add a note.